src/main/java/com/crudtest/service/user/impl/DefaultUserService.java

```diff
@@ -16,20 +19,17 @@
     private UserRepository userRepository;
 
     List<User> users = new ArrayList<User>();
-    @Override
-    public User getUserByLastName(String lastName) {
-        return userRepository.findByLastName(lastName);
-    }
+
 
     @Override
     public void saveUser(User user) {
```

Comment:
This method should contain the logic for setting the Date attributes as required.